Source/WebCore/PAL/pal/spi/cocoa/PassKitSPI.h

```diff
@@ -357,6 +342,23 @@ typedef void(^PKCanMakePaymentsCompletion)(BOOL isValid, NSError *);
 @end
 #endif
 
+#if HAVE(PASSKIT_APPLE_PAY_LATER_AVAILABILITY)
+
+// FIXME: rdar://107955442 Remove staging code.
+
+typedef NS_ENUM(NSUInteger, PKApplePayLaterAvailability);
+
+#define PKApplePayLaterAvailable 0
+#define PKApplePayLaterUnavailableMerchantIneligible 1
+#define PKApplePayLaterUnavailableItemIneligible 2
+#define PKApplePayLaterUnavailableRecurringTransaction 3
```

Comment:
Let's move these into `WebPaymentCoordinatorProxyCocoa.mm`. As we found out last week, it isn't safe to have `#define`s here, due to the import at the bottom of this file.

Reply:
Yup. Please check https://github.com/WebKit/WebKit/commit/b33f3e1e9b538e099e3121818de80238325c2e73